### DISCUSSION TITLE
Wishing Well pending rewards reminder

### DISCUSSION THREAD

**Discussion by dfvgbh** [sunflower-land/sunflower-land#1149] [Ideas]
Please, make some notification on the main screen to remind farmers that there are pending rewards in wishing well.

Currently, it's really inconvenient to put some 3d timer not to forget to collect rewards.

And it would be much better to make the process of creating a new wish automatically after the 3d period!

**Comment by Polysemouse** [comment]
This somewhat aligns with my proposal (https://github.com/sunflower-land/sunflower-land/discussions/1095) where you can see all your countdowns in your farm in a single place including the wishing well
